drools-wb-services/drools-wb-verifier/drools-wb-verifier-backend/src/test/java/org/drools/workbench/services/verifier/webworker/client/fromfile/DecisionTableAnalyzerFromFileTest.java

```diff
@@ -142,6 +142,24 @@ public void testFile3() throws
     }
 
     @Test
+    public void testLHSConflictsArePickedUpForEachFieldOfAPattern() throws
+            Exception {
+        final String xml = loadResource("GUVNOR-3513.gdst");
+
+        final GuidedDecisionTable52 table52 = GuidedDTXMLPersistence.getInstance()
+                .unmarshal(xml);
+
+        final Analyzer analyzer = analyzerProvider.makeAnalyser(table52);
+
+        analyzer.resetChecks();
+        analyzer.analyze();
+
+        analyzerProvider.getAnalysisReport().stream().forEach(i->System.out.println(i.getCheckType()));
```

Comment:
I think `System.out.println(i.getCheckType()` is really useful during fixing/investigating some bug, however then the test is just run automatically by a machine, without checking the output. Is there any chance for some debug level log or some assertion instead of the `System.out.println`?

Reply:
We can probably make a method that checks if the report is empty and then we only print the content if the test fails.

Reply:
This sounds acceptable to me. thanks